dtfs-central-api/api-tests/v1/utilisation-reports/put-keying-data-mark-as-done.api-test.ts

```diff
@@ -36,7 +43,7 @@ describe(`PUT ${BASE_URL}`, () => {
   it('returns a 400 when the fee record ids are not a valid ids', async () => {
     // Arrange
     const requestBody = {
-      user: MOCK_TFM_USER,
+      user: tfmSessionUser,
```

Comment:
Could this just be the following?:

```ts
user: aTfmSessionUser(),
```

instead of having to add `let tfmSessionUser: TfmSessionUser;` and a `beforeEach`

Reply:
The reason I did it this way was to access parameters in tests without having to constantly redeclare the `const tfmSessionUser = aTfmSessionUser()`, but its only used once in this file -- makes accessing parameters a lot easier ie line 146 (only here for now, but generally makes this more scalable as apposed to adding the same line of  `const tfmSessionUser = aTfmSessionUser()` to every test )